aospy/calc.py

```diff
@@ -767,3 +767,19 @@ def load(self, dtype_out_time, dtype_out_vert=False, region=False,
         if plot_units:
             data = self.var.to_plot_units(data, dtype_vert=dtype_out_vert)
         return data
+
+    def addAttrs(self, data):
+        if (isinstance(data, xr.DataArray)):
+            self.addAttrsDataArray(data)
+        else:
+            for name, da in data.data_vars.items():
+                self.addAttrsDataArray(da)
+
+    def addAttrsDataArray(self, data):
```

Comment:
Similar to the above comment regarding function naming, I recommend calling this one `add_units_and_description_attrs_da`.